tripal_chado/includes/TripalImporter/FASTAImporter.inc

```diff
@@ -536,7 +535,7 @@ class FASTAImporter extends TripalImporter {
         }
 
         // Get the relationship subject
-         $subject = "";
+         $subject = $name ? $name : $uname;
```

Comment:
should this actually be $uname or null?

Reply:
I think it should be 
```suggestion
         $subject = $uname ? $uname : "";
```
based on https://github.com/tripal/tripal/pull/738/files#diff-8006ac719a97e82d92833b60886802a2L831 where the query looking up the parent/subject for the relationship checks the uniquename column.